runtime/near-vm-runner/RUNTIMES.md

```diff
@@ -0,0 +1,218 @@
+This document describes the considerations that go into picking a Wasm runtime as well as the
+conclusions we made after evaluating a number of VM backends and frontends.
+
+# Criteria
+
+A number criteria have been used to evaluate a Wasm runtime for use in the implementation of the
+NEAR protocol. Some of the criteria are already listed in the [FAQ] document and this document
+gives a more thorough look. Listed roughly in the order of importance:
+
+* security – how well does the runtime deal with untrusted input;
```

Comment:
"how well" is somewhat vague, we'd better define what are exact criteria here: i.e. have ability to limit resource consumption, trap on incorrect input, etc.

Reply:
Right the requirement section lists some of the… requirements with regards to security but I definitely missed e.g. resource consumption aspect. Will add that in.